### DISCUSSION TITLE
Keyboard shortcut to save document on `Ctrl/Cmd + S`

### DISCUSSION THREAD

**Discussion by yandodov** [payloadcms/payload#1812] [Feature Requests & Ideas]
Due to my prior [Kirby CMS](https://getkirby.com/) usage, I've developed a habit of pressing <kbd>Ctrl + S</kbd> to save changes. It'd be neat to have this in Payload as well. Right now, pressing <kbd>Ctrl + S</kbd> prompts you to save the HTML of the admin, which is useless.

**Comment by DanRibbens** [comment]
This would make a good pull request to merge if somebody wants to add it. 

Are there other hotkeys editors might want?

**Reply by rpfaeffle** [reply]
I've created a PR in #1821.
